assets/generated_samples_list.json

```diff
@@ -472,6 +472,38 @@
     "title": "Authenticate with token",
     "key": "authenticate_with_token"
   },
+  "change_viewpoint": {
+    "category": "MapView",
+    "description": "Set the map view to a new viewpoint.",
+    "formal_name": "ChangeViewpoint",
+    "ignore": false,
+    "images": [
+      "change_viewpoint.jpg"
+    ],
+    "keywords": [
+      "animate",
+      "extent",
+      "pan",
+      "rotate",
+      "scale",
+      "view",
+      "zoom"
+    ],
+    "offline_data": [],
+    "redirect_from": [],
+    "relevant_apis": [
+      "Geometry",
+      "Map",
```

Comment:
it should be `ArcGISMap`, the file may need to regenerate after modifying the README.